Add documentation for handling failed simulations (#135)
OpenFreeEnergy/IndustryBenchmarks2024: docs/source/public/overview.rst

Please reach out to the openfe team if you have any questions on how to adapt this script to your internal needs, we would be happy to assist with this.

Handling failed edges
=====================

It is possible that some of the simulations in the network fail.
More specifically, not all repeats of an edge may finish successfully, or an edge can fail entirely.

> We propose the following strategy for dealing with those failures:

Reviewer: ```suggestion
You should follow this strategy for dealing with those failures:
```
